v1/telemetry-worker.js

```diff
@@ -0,0 +1,62 @@
+importScripts("telemetry.js", "big.js")
+
+var gActions = {
+  "Histogram-count": function() {
```

Comment:
use the same style as the other files, we don't quote function names or use hyphens inside the names